.github/workflows/ci.yml

```diff
@@ -11,20 +11,27 @@ env:
   DOTNET_CLI_TELEMETRY_OPTOUT: true
   DOTNET_CONSOLE_ANSI_COLOR: true
 
+defaults:
+  run:
+    shell: bash
+
 jobs:
   GetVersion:
-    runs-on: ubuntu-20.04
+    runs-on:
+       group: OpenTAP-SpokeVPC
+       labels:  [Linux, X64]
+    container: ghcr.io/opentap/oci-images/build-dotnet:latest
     outputs:
       ShortVersion: ${{ steps.gitversion.outputs.ShortVersion }}
       GitVersion: ${{ steps.gitversion.outputs.GitVersion }}
     steps:
       # The get-gitversion action installs OpenTAP and fetches with fetch-depth: 0
       - name: GitVersion
         id: gitversion
-        uses: opentap/get-gitversion@v1.1
+        uses: opentap/get-gitversion@main
 
   Build:
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, Windows]
```

Comment:
Can we switch this back to Linux runners?

Reply:
restored back